lib/converters/v1.0.0/converter-v1-to-v2.js

```diff
@@ -288,6 +288,10 @@ _.assign(Builders.prototype, {
         if (!_.isEmpty(requestV1.graphqlModeData)) {
             graphqlModeData = requestV1.graphqlModeData;
         }
+        else if (requestV1?.dataOptions?.graphql !== undefined && !_.isEmpty(requestV1.data)) {
```

Comment:
Can we avoid using conditional chaining? Not all consumers of this would have support of it.

Reply:
Done.